GPT/gpt.py

```diff
@@ -110,6 +110,14 @@ def paste_model_confirmation_gui():
         GuiState.text_to_confirm = ""
         confirmation_gui.hide()
 
+    def paste_selected(result: str):
```

Comment:
Isn't `paste_and_select` perhaps more clear?

Reply:
Works for me 👍